composer.json

```diff
@@ -60,7 +60,8 @@
             "source-directory": "config"
         },
         "config-plugin": {
-            "bootstrap": "bootstrap.php"
+            "bootstrap": "bootstrap.php",
+            "widgets": "widgets.php"
```

Comment:
Maybe set it as directory instead of file, for config with many widgets?

Reply:
This is only configuration of package. In app you can use configs from directory.